modules/service/src/main/resources/graphql/itc.graphql

```diff
@@ -14812,7 +14812,7 @@ input InstrumentModesInput {
 }
 
 # Params for spectroscopy integration time calculation
-input SpectroscopyIntegrationTimeInput {
+input SpectroscopyInput {
```

Comment:
The input is for both modes

Reply:
Having `Time` in the name differentiated it from `TimeAndGraphs`.